### PR TITLE
added delete validation

### PR DESCRIPTION
**Pull request by fardinzam**
no entries selected:
Screenshot 2023-10-26 at 11 25 43 AM

entries selected:
Screenshot 2023-10-26 at 11 26 06 AM

### CODE REVIEW

File: src/components/dynamic/admin/dashboards/Toolbar.jsx [hunk 1]
```diff
@@ -139,12 +139,17 @@ const Toolbar = ({
...
         <FaTrashAlt
           data-cy="delete"
-          onClick={() =>
-            setPopup({
-              ...popup,
-              visible: true,
-            })
-          }
+          onClick={() => {
+            const numSelectedObjects = objects.filter((a) => a.selected).length;
```
Comment (shahdivyank) [inline src/components/dynamic/admin/dashboards/Toolbar.jsx]: this should go into a onDelete handler

File: src/components/dynamic/admin/dashboards/Toolbar.jsx [hunk 2]
```diff
@@ -139,12 +139,17 @@ const Toolbar = ({
...
+            if (numSelectedObjects) {
```
Comment (shahdivyank) [inline src/components/dynamic/admin/dashboards/Toolbar.jsx]: should be an explicit number check such as `objects.filter((a) => a.selected).length === 0`

File: src/components/dynamic/admin/dashboards/Toolbar.jsx [hunk 3]
```diff
@@ -139,12 +139,17 @@ const Toolbar = ({
...
+              setPopup({
+                ...popup,
+                visible: true,
+              });
+            } else {
+              toast("❌ Select row(s) before pressing the delete button");
```
Comment (shahdivyank) [inline src/components/dynamic/admin/dashboards/Toolbar.jsx:144]: u can remove the else statement, the if statement should only be triggering toasts and returning from the function. check other functions that have a toast to see how they handle the error handling

**Comment by shahdivyank** [comment]
theres a bunch of extra files here

**Comment by fardinzam** [comment]
i'm closing this pr

**Pull request closed without merging**